pkg/apis/core/v1/defaults.go

```diff
@@ -69,6 +70,14 @@ func SetDefaults_Volume(obj *v1.Volume) {
 			EmptyDir: &v1.EmptyDirVolumeSource{},
 		}
 	}
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImageVolume) && obj.Image != nil && obj.Image.PullPolicy == "" {
+		// PullPolicy defaults to Always if :latest tag is specified, or IfNotPresent otherwise.
+		if strings.HasSuffix(obj.Image.Reference, ":latest") {
```

Comment:
use the same `parsers.ParseImageName` as above in this file please.

Reply:
Done